authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,11 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+from django.conf import settings
```

Comment:
You can consider removing unused imports and also following PEP8 conventions, we leave a line at the end of a file.